ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/LedgerTestSuite.scala

```diff
@@ -26,6 +27,16 @@ private[testtool] abstract class LedgerTestSuite(val session: LedgerSession) {
 
   protected final implicit val ec: ExecutionContext = session.executionContext
 
+  // TODO Make this configurable
+  final private[this] val eventually = RetryStrategy.exponentialBackoff(10, 10.millis)
+
+  final def assertEventually[A](message: String)(assertion: () => Future[A]): Future[Unit] =
```

Comment:
Could also be a by-name parameter, which gives a bit nicer syntax on use site.

Reply:
Yeah, for some reason I thought _by-name_ parameters were evaluated only once but I guess it shows that I wrote this code on a Friday afternoon. :slightly_smiling_face:

Reply:
Also `message` is unused (it was part of an initial experiment), I've reworked the whole thing a bit, thanks for the feedback.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/2900/commits/2cef0bf1f45303a27f3e2980c106e6e6694627b6